cinn/ir/ir_schedule.cc

```diff
@@ -1374,7 +1427,196 @@ void IRSchedule::CopyTransformAndLoopInfo(const Expr& block, const Expr& block_t
   VLOG(3) << "res is : " << res;
   std::vector<Expr> all_loops = this->GetLoops(block);
   CHECK(!all_loops.empty());
-  helper_.Replace(all_loops[0], res);
+  this->Replace(all_loops[0], res);
+}
+
+IRSchedule::IRSchedule() {}
+
+IRSchedule::IRSchedule(IRSchedule&& other) : impl_(std::move(other.impl_)), trace_(other.trace_) {}
```

Comment:
Does `other.trace_` need to be moved?

Reply:
not necessary, but I will add this one to be more stable